README

```diff
@@ -72,14 +85,27 @@ specified.
 
 CacheOrBust uses autotools, which can be installed through your
 distribution's package manager, and depends on libraries and headers from
-Kyoto Cabinet and Kyoto Tycoon, which can be found at http://fallabs.com/.
+Kyoto Cabinet (http://fallabs.com/kyotocabinet/) and Kyoto Tycoon (http://fallabs.com/kyototycoon/).
+
+On CentOS 7 the following packages should be installed first:
+
+    $ sudo yum install autoconf make automake gcc gcc-c++ libtool kyototycoon-devel kyotocabinet-devel
+    NOTE: The kyoto*devel packages are in the Magnetic RPM repository
 
 With dependencies installed, you can configure the build environment with:
 
-    $ autoreconf
-    $ ./configure
+    $ autoreconf --force --install
+    $ sed -i -e 's/ -g0 -O2//' -e 's/ -O0//' configure
+    $ sed -i -e 's/ -g0 -O2//' -e 's/ -O0//' Makefile.am
+    $ ./configure --prefix=/usr
 
 And then build the CacheOrBust shared object library with:
 
     $ make
     $ [sudo] make install
+
+Sample run command:
+
+    $ ktserver -ls -port 6000 -plsv ./.libs/ktcacheorbust-1.0.so \
```

Comment:
what's the difference between these two port references? one for kyoto, one for cacheorbust?

Reply:
Correct